### ISSUE TITLE
Unreachable else statement

### ISSUE THREAD

**Issue by kvarys** [yycdavid/program-synthesis-guided-RL#3]
Dear Researchers,

In modular_ac.py around line 120 there is the following if statement which always gets executed, making the else unreachable:

`
probs = np.exp(log_probs.detach().cpu().numpy())
                                                             
for j, (pr, i) in enumerate(zip(probs, indices)):                                                            
    if self.training or True:                                                                                
        a = np.random.choice(self.n_actions, p=pr)                                                           
    else:                                                                                                    
        # print("Choose most probable action")                                                               
        a = np.argmax(pr)
`
Perhaps would be worth removing it?

Kind regards!
